lib/Backend/Inline.cpp

```diff
@@ -2442,6 +2442,14 @@ IR::Instr * Inline::InlineApplyWithArgumentsObject(IR::Instr * callInstr, IR::In
     IR::Instr* builtInStartInstr;
     InsertInlineeBuiltInStartEndTags(callInstr, 3, &builtInStartInstr); //3 args (implicit this + explicit this + arguments = 3)
 
+    // Move argouts close to call. Globopt expects this for arguments object tracking.
```

Comment:
No, this has nothing to do with OOP jit. This got inadvertently introduced over 2 months ago.